omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js

```diff
@@ -64,39 +65,89 @@ class SessionView extends Component {
 
 
     componentWillMount() {
+
         this.setState(() => {
 
             let sessionData = this.props.courseSessions[this.props.computedMatch.params.course_id][this.props.computedMatch.params.session_id]
             let courseData = this.props.courses[this.props.computedMatch.params.course_id]
-
             let instructorData = this.props.instructors
 
 
-
             return {
                 sessionData: sessionData,
-                courseData: courseData
+                courseData: courseData,
+                instructorData: instructorData
             }
         })
-
-
     }
 
-    render() {
 
-        const flexContainer = {
-            display: "flex",
-            justifyContent: "flex-start",
-            width: "50%",
-            padding: 0,
+
+    // Formatting the date in the view
+    formatDate = (start, end) => {
+        let DayConverter = {
+            1: "Monday",
+            2: "Tuesday",
+            3: "Wednesday",
+            4: "Thursday",
+            5: "Friday",
+            6: "Saturday",
         };
+        let MonthConverter = {
+            0: "January",
+            1: "Febuary",
+            2: "March",
+            3: "April",
+            4: "May",
+            5: "June",
+            6: "July",
+            7: "August",
+            8: "September",
+            9: "October",
+            10: "November",
+            11: "December"
+        }
+
+        let date = new Date(start)
+        let dateNumber = date.getDate()
+        let dayOfWeek = date.getDay()
+        let startMonth = date.getMonth()
+        // Gets days
+        let Days = [dayOfWeek]
+        Days = Days.map((day) => {
+            return DayConverter[day];
```

Comment:
As shorthand, this `map` can be written as `Days = Days.map((day) => DayConverter[day]);` If the part after the `=>` is not a function, then whatever its value is becomes the return value automatically, as if it were written in the expanded form you have.

Also, this seems to be a map on a single element array. It can be replaced by simply `let Days = DayConverter[dayOfWeek];`